config/styleguide.config.js

```diff
@@ -65,26 +75,19 @@ module.exports = {
       content: path.resolve('docs/intro/welcome.md'),
     },
     {
-      name: 'Foundational elements',
+      name: 'Component Catalogue',
```

Comment:
I suggest `Components & Utilities` or `TDS Packages` since technically this list contains React Component packages and SCSS packages.